packages/schematics/angular/service/schema.json

```diff
@@ -11,6 +11,10 @@
       "type": "string",
       "default": "app"
     },
+    "appRoot": {
+      "type": "string",
+      "default": "app"
```

Comment:
We shouldn't have a default to ensure backwards compatibility with CLI versions that aren't sending the appRoot options.

Reply:
done.